_tests/auth_test.go

```diff
@@ -12,10 +12,10 @@ var _ = Describe("Auth", func() {
 		})
 
 		It("won't print the current user", func() {
-			output, err := execute("deis auth:whoami")
-			Expect(err).To(HaveOccurred())
-			Expect(output).To(ContainSubstring("Not logged in."))
-			Expect(output).NotTo(ContainSubstring(testUser))
+			Expect(execute("deis auth:whoami")).To(BeASuccessfulCmdWithOutput(
```

Comment:
`BeASuccessfulCmdWithOutput` seems odd here, considering that the command should expect to have exited non-zero with the expected substring.